controllers/example_controller/keyboard_controller.py

```diff
@@ -0,0 +1,72 @@
+from sr.robot import Robot
+from controller import Keyboard
+
+TIMESTEP = 16
+NO_KEY_PRESSED = -1
+
+
+def print_distance_sensors(robot: Robot) -> None:
+    distance_sensor_names = [
```

Comment:
Should we just import this instead?

Reply:
I considered that, however they're not the same names.